src/styles/blocks/card.css

```
@@ -0,0 +1,88 @@
+.card--container {
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: #fff;
+}
+
+.card__photo {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
+}
+
+.card__desc {
+  margin: 0 16px 4px;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__product-code {
+  margin: 0 0 16px 16px;
+  font-family: "Roboto", sans-serif;
```

You can set the font for the whole page and inherit it where needed not to duplicate font-family everywhere
```suggestion
  font-family: inherit;
```